koku/reporting/provider/ocp/models.py

```diff
@@ -956,3 +959,162 @@ class Meta:
     cost_model_memory_cost = models.DecimalField(max_digits=33, decimal_places=15, null=True)
     cost_model_volume_cost = models.DecimalField(max_digits=33, decimal_places=15, null=True)
     cost_model_rate_type = models.TextField(null=True)
+
+
+class OCPGpuSummaryP(models.Model):
+    """A summarized partitioned table for GPU cost queries.
+
+    This table gives a daily breakdown of GPU costs aggregated from daily_summary.
+    """
+
+    class PartitionInfo:
+        partition_type = "RANGE"
+        partition_cols = ["usage_start"]
+
+    class Meta:
+        """Meta for OCPGpuSummaryP."""
+
+        db_table = "reporting_ocp_gpu_summary_p"
+        indexes = [
+            models.Index(fields=["usage_start"], name="ocpgpusumm_usage_start"),
+            models.Index(fields=["cluster_id"], name="ocpgpusumm_cluster_idx"),
+            models.Index(fields=["namespace"], name="ocpgpusumm_namespace_idx"),
+            models.Index(fields=["node"], name="ocpgpusumm_node_idx"),
+            models.Index(fields=["gpu_vendor_name"], name="ocpgpusumm_vendor_idx"),
+            models.Index(fields=["gpu_model_name"], name="ocpgpusumm_model_idx"),
+        ]
+
+    id = models.UUIDField(primary_key=True)
+    cluster_id = models.TextField()
+    cluster_alias = models.TextField(null=True)
+    namespace = models.CharField(max_length=253, null=True)
+    node = models.CharField(max_length=253, null=True)
+    pod = models.CharField(max_length=253, null=True)
```

Comment:
Are we showing pod level information in the UI mocks for this? This column doesn't feel needed, but the mocks are kind of hard to track so feel free to correct me if I am wrong here.